### DISCUSSION TITLE
Retweet 5.000 puan

### DISCUSSION THREAD

**Discussion by Bulent3407** [IBAX-io/go-ibax#1724] [Soru Cevap - Türkçe] [answered]
1 saattir deniyorum retweet yapmama rağmen 5.000 puanı alamıyorum

**Comment by Abdullahoda** [comment, marked as answer]
yoğunluktan dolayı ilerleyen saatlerde verilecektir..

**Reply by Bulent3407** [reply]
Beklemedeyiz

**Reply by alisabanoglu** [reply]
evet bende yarım saatir bilgisayar cem telefonu deniyorum olmuyor....:((((

**Reply by coinsinov** [reply]
Benim Takibe 2000 puanımı da yediler. :) Bu 5000'de zor gelecek gibi.

**Reply by hsyntpl** [reply]
Son gün s*ctılar işin içine👏👏👏

**Comment by ensar74** [comment]
bence sınırlı kişiler aldı ondan kitlediler gibi

**Reply by Bulent3407** [reply]
100 kişi yapmış

**Comment by hsyntpl** [comment]
Naptınız, Bi türlü olmuyor

**Reply by Bulent3407** [reply]
Olmuyor

**Comment by erish54** [comment]
olmuyor maalesef...

**Reply by Bulent3407** [reply]
Olmuyor
